applications/skyepass.md

```diff
@@ -0,0 +1,251 @@
+| 2.     | Smart Contract                 | All 4 smart contracts mentioned in the `Ecosystem Fit` section. For this phase, these contracts are testing contracts, and they are subject to minor changes as long as they provide the same/very similar functionality. |
+| 3.     | UI/UX Design Standards         | Color Schema, Composition & Balance, Typography, Animation, Navigation, Icon Set, Brand Logo |
+| 4.     | Basic Marketing Efforts        | Get on product-hunt and send out private alpha invitation    |
+| 5.     | Testing                        | Well... test code and basic internal security audit          |
+### Milestone 2  — Ready for Beta Release
+
+* **Estimated Duration:** 6 Weeks
+
+* **FTE:**  4
+
+* **Costs:** 3,400 DAI is what we are requesting for this phase. We understand that there is a $30,000 limit and we will fill in the funding gap in other ways. 
+
+* **Actually Costs**: 35,920 DAI (4 FTE * 35 Hours per week * 6 Weeks * \$38Hourly Wage + A security Auditing \$4,000 ) We will be far less stressed at this phase. It's gonna be more about tweaking here and there and security auditing. 
+
+    
+
+| Number | Deliverable                                    | Specification                                                |
+| ------ | ---------------------------------------------- | ------------------------------------------------------------ |
+| 0a.    | Developer Resource/API Documentation/Community | - A comprehensive API spec documentation <br/>- Riot group for support, suggestions and questions<br/> |
+| 0b.     | **Security Auditing**                          | Audited by a trusted 3rd party                               |
+| 1.     | Desktop App/Browser Extension                  | Create an open Github repo for `extensions`, build a management system for open PR of new integrations. The "marketplace" in the desktop app will pull a list of available integrations from the repo.  <br/>Support at least 2 password importing source<br/>Support unencrypted password exporting<br/>Add in support for browser extension to inject hot wallet like Metamask<br/> |
+| 2.     | Smart Contracts                                | - Portals to interact with smart contract 3 & 4. Either standalone webpages or built into the desktop apps. <br/>- All contract tested and audited and we will deploy the V1.0 contract to the appropriate parachains. |
+| 3.     | Partnership                                    | Build partnerships for developer/marketing/user support resources |
+| 4.     | Mobile Apps                                    | Draft up UI/UX designs for mobile apps.                      |
+| 5.     | Equity Fundraising Prep                        | Draft up material in preparation of an institutional fundraising |
```

Comment:
Feel free to remove this delivery, since we can not really test it anyway. Also we are mostly interested in the development of open source software that can be used by others and not in the business side of your project/application.

Reply:
Done.